TestSwitchApi/TestSwitchApi/Models/DataModels/AdminUserSession.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace TestSwitchApi.Models.DataModels
+{
+    public class AdminUserSession
+    {
+        [Key]
+        public string Id { get; set; }
+        public int AdminUserID { get; set; }
+        public string SessionStart { get; set; }
+        public string SessionEnd { get; set; }
```

Comment:
As mentioned in the DB file, I think we can store these as their actual types a little more - so `UUID` and `DateTime`.
That should reduce the amount of annoying translating back and forth between strings.

Reply:
I've updated the types for this.